examples/filesender/main.go

```diff
@@ -49,7 +50,7 @@ func main() {
 		APISecret:           apiSecret,
 		RoomName:            roomName,
 		ParticipantIdentity: identity,
-	}, &lksdk.RoomCallback{}, lksdk.WithAutoSubscribe(false))
+	}, &lksdk.RoomCallback{}, lksdk.WithAutoSubscribe(false), lksdk.WithICETransportPolicy(webrtc.ICETransportPolicyRelay))
```

Comment:
maybe comment out from the example so it doesn't require relay?

Reply:
Yes! Oops nice catch 

Neil is doing this for a specific customer, not sure how this will land yet